src/models/Card.js

```diff
@@ -96,7 +96,7 @@ function save(next) {
         if (!this.isMandatory) this.isMandatory = false;
         break;
       case ORDER_THE_SEQUENCE:
-        if (!this.orderedAnswers) this.orderedAnswers = [{ text: '' }, { text: '' }];
+        if (!this.orderedAnswers) this.orderedAnswers = [{ text: '' }, { text: '' }, { text: '' }];
```

Comment:
ESt-ce que `if (!this.orderedAnswers)` est necessaire ?

Reply:
je ne sais plus pourquoi on vérifie ça avant chaque création, dans le doute je le garderai

Reply:
J'ai verifié et on fait ces verifs car dès qu'on fait un `create`, un `findOneAndUpdate` (et peut etre d'autres operations), un document.save() est appelé, ce qui déclenche le middleware associé. 

Si on ne met pas ces verifs, ca veut dire qu'on écraserait les infos passées en payload par celles définies dans la fonction `save`

Reply:
donc il faut le garder ?

Reply:
oui